policyengine_us/variables/gov/irs/credits/estate/basic_exclusion_amount.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_us.model_api import *
+
+
+class basic_exclusion_amount(Variable):
```

Comment:
```suggestion
class estate_credit_basic_exclusion_amount(Variable):
```

Reply:
adjust label and file title accordingly - same for other estate credit variables